Dockerfile

```diff
@@ -41,13 +39,12 @@ WORKDIR /tmp
 RUN pip install --no-cache-dir -r requirements.txt
 
 # Install frontend dependencies using NPM
-COPY package.json /opt/npm/
+COPY package.json package-lock.json /opt/npm/
 
 # Switch to /opt/npm to install dependencies outside home dir
 WORKDIR /opt/npm
 RUN npm install && \
-    chown -R 10001:10001 /opt/npm && \
-    npm cache clean
+    chown -R 10001:10001 /opt/npm
 
```

Comment:
NPM now raises an error when manually cleaning up since it prevents the caching to work effectively.